concourse/pipelines/create-cloudfoundry.yml

```
@@ -4396,7 +4396,7 @@ jobs:
                 cf enable-service-access elasticsearch -b aiven-broker
 
                 # Disable deprecated plans
-                cat <<EOF | xargs -n1 cf disable-service-access elasticsearch -p
+                cat <<EOF | xargs -n1 cf disable-service-access -b elasticsearch -p
```

I think the command needs to be slightly different as `-b` specifies the name of the broker: `cf disable-service-access SERVICE [-b BROKER] [-p PLAN] [-o ORG]`.

```suggestion
                cat <<EOF | xargs -n1 cf disable-service-access elasticsearch -b aiven-broker -p
```